src/components/tx/TxDetails/show.js

```diff
@@ -1,21 +1,26 @@
+/* @flow */
 import React from 'react';
 import PropTypes from 'prop-types';
 import { connect } from 'react-redux';
 import FlatButton from 'material-ui/FlatButton';
-
-import AccountAddress from 'elements/AccountAddress';
-import AddressAvatar from 'elements/AddressAvatar';
-import { gotoScreen } from 'store/screenActions';
-import { toNumber, toDate } from 'lib/convert';
-import IdentityIcon from 'elements/IdentityIcon';
-import { Form, styles } from 'elements/Form';
+import { convert } from 'emerald-js';
+
+import AccountAddress from '../../../elements/AccountAddress';
+import AddressAvatar from '../../../elements/AddressAvatar';
+import { gotoScreen } from '../../../store/screenActions';
+import { toDate } from '../../../lib/convert';
+import IdentityIcon from '../../../elements/IdentityIcon';
+import { Form, styles, Row } from '../../../elements/Form';
 import TxStatus from './status';
+import { Currency } from '../../../lib/currency';
```

Comment:
How come `/lib/` is no longer global dir?

Reply:
Jest test framework can't work if we import like a global module.
May be there is solution...